scripts/run-integration-tests

```diff
@@ -42,33 +41,36 @@ CLEANUP_TESTS=${CLEANUP_TESTS:-0}
 PROXY_PORT=${PROXY_PORT:-4200}
 PROXY_PID=0
 
-PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
+PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)"
 CYPRESS_FOLDER="cypress"
 TESTS_FOLDER="tests"
 
 PLUGINS_PATH=${PLUGINS_PATH:-''}
 
-while getopts 's:hnr:?' flag; do
+while getopts 's:hknr:?' flag; do
   case "${flag}" in
-    s) SEARCH=${OPTARG:-'*'} ;;
-    h) CYPRESS_HEADED="true" ;;
-    n) CYPRESS_NO_EXIT="true" ;;
-    r) RETRIES=${OPTARG} ;;
-    ?)
-      echo -e "+-------------------------------+"
-      echo -e "| DC/OS Integration test runner |"
-      echo -e "+-------------------------------+\n"
-      echo -e "Don't forget running npm run build:assets first!\n\n"
-      echo -e "Usage: \n"
-      echo -e "\tnpm run test:integration -- [-s <FILE_NAME>] [-hn] [-r <RETRIES>]\n"
-      echo -e "Options:\n"
-        printf "  %-17s %-30s \n" "-s <TestFileName>" "Run single Test File. All files if omitted"
-        printf "  %-17s %-30s \n" "-h" "Run Chrome in Headed mode"
-        printf "  %-17s %-30s \n" "-n" "Don't close Chrome after tests are finished"
-        printf "  %-17s %-30s \n" "-r <NUM_RETRIES>" "Number of retries, 3 by default"
-        printf "  %-17s %-30s \n" "-?" "Prints this help"
-        echo ""
-      exit 0 ;;
+  s) SEARCH=${OPTARG:-'*'} ;;
+  h) CYPRESS_HEADED="true" ;;
+  n) CYPRESS_NO_EXIT="true" ;;
+  k) NO_BUILD="true" ;;
+  r) RETRIES=${OPTARG} ;;
+  ?)
+    echo -e "+-------------------------------+"
+    echo -e "| DC/OS Integration test runner |"
+    echo -e "+-------------------------------+\n"
+    echo -e "Don't forget running npm run build:assets first!\n\n"
+    echo -e "Usage: \n"
+    echo -e "\tnpm run test:integration -- [-s <FILE_NAME>] [-hnk] [-r <RETRIES>]\n"
+    echo -e "Options:\n"
+    printf "  %-17s %-30s \n" "-s <TestFileName>" "Run single Test File. All files if omitted"
+    printf "  %-17s %-30s \n" "-h" "Run Chrome in Headed mode"
+    printf "  %-17s %-30s \n" "-n" "Don't close Chrome after tests are finished"
+    printf "  %-17s %-30s \n" "-r <NUM_RETRIES>" "Number of retries, 3 by default"
+    printf "  %-17s %-30s \n" "-k" "Don't run the build, just assume something runs on the right port"
```

Comment:
what about `-d` which does `npm start` in the background? 🤔

Reply:
I think there could be different usecases, but I would also be fine with that. How would you handle the output?